src/Bridge/Symfony/Bundle/Resources/config/api.xml

```diff
@@ -44,6 +44,8 @@
             <tag name="routing.loader" />
         </service>
 
+        <service id="ApiPlatform\Core\Api\UrlGeneratorInterface" alias="api_platform.router" public="true" />
```

Comment:
`public="true"` should be removed (there is no reason to make this alias public).